pkg/virt-handler/device-manager/mediated_device.go

```diff
@@ -417,7 +417,15 @@ func getMdevTypeName(mdevUUID string) (string, error) {
 	// #nosec No risk for path injection. Path is composed from static base  "mdevBasePath" and static components
 	rawName, err := ioutil.ReadFile(filepath.Join(mdevBasePath, mdevUUID, "mdev_type/name"))
 	if err != nil {
-		return "", err
+		if os.IsNotExist(err) {
```

Comment:
@xphyr perhaps you could mention in a comment above that `name` file is optional in this interface.
`'name', 'description' and other vendor driver specific attributes are optional.`[1]

[1] https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/ABI/testing/sysfs-bus-vfio-mdev

Reply:
@vladikr are you looking for a comment in the code, or just in the PR request?

Reply:
also, how standard is it to derive the mdev name by following a symbolic link and using the file name?

Reply:
That's the standard. In the Nvidia case, it is so as well. It would point to the type name, for example: `nvidia-222`, we could work with these names but Nvidia provides an optional name (more meaningful) in the `name` file (`nvidia-222` = `GRID T4-1B`)

Reply:
With an Intel GPU it looks like this:
sh-4.4# pwd
/sys/bus/mdev/devices/973069b7-2025-406b-b3c9-301016af3150
sh-4.4# ls -lsa
total 0
0 drwxr-xr-x.  4 root root    0 Mar 12 14:37 .
0 drwxr-xr-x. 12 root root    0 Mar 12 14:36 ..
0 lrwxrwxrwx.  1 root root    0 Mar 12 14:37 driver -> ../../../../bus/mdev/drivers/vfio_mdev
0 drwxr-xr-x.  2 root root    0 Mar 12 14:37 intel_vgpu
0 lrwxrwxrwx.  1 root root    0 Mar 12 14:37 iommu_group -> ../../../../kernel/iommu_groups/9
0 lrwxrwxrwx.  1 root root    0 Mar 12 14:37 mdev_type -> ../mdev_supported_types/i915-GVTg_V5_8
0 drwxr-xr-x.  2 root root    0 Mar 12 14:37 power
0 --w-------.  1 root root 4096 Mar 12 19:00 remove
0 lrwxrwxrwx.  1 root root    0 Mar 12 14:37 subsystem -> ../../../../bus/mdev
0 -rw-r--r--.  1 root root 4096 Mar 12 14:37 uevent